Expensify/App
# Installs Onyx from the react-native-onyx repo

~Held on https://github.com/Expensify/React-Ion/pull/1~

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/145302

### Tests
1. Run `npm i && npm run web`
1. Make sure the app builds and works
1. On web, open up the chat app in multiple tabs. Verify that the actions are synced across the tabs
